.eslintrc

```diff
@@ -8,6 +8,7 @@
     "ActiveXObject": true
   },
   "rules": {
-    "import/no-amd": 0
+    "import/no-amd": 0,
+    "no-param-reassign": 0
```

Comment:
I should have brought this up, apologies for that.  I'll do so at standup tomorrow